packages/core/src/components/ui/ProductGallery/ProductGallery.tsx

```diff
@@ -212,14 +219,15 @@ function ProductGallery({
             </div>
           )}
           {/* Render ALL products */}
-          {productGalleryData ? (
+          {hasFacets ? (
```

Comment:
Are you not changing the logic here?

Reply:
We're checking the the facets existence now. Previously we use data from useSWR as `productGalleryData` alias. now we don't have the same `data` and we are checking the facets.